src/components/collapsible-panel.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import { caretDownIcon, caretRightIcon } from '@jupyterlab/ui-components';
+
+export interface ICollapsiblePanelProps {
+  title: JSX.Element | string;
```

Comment:
Do we want to allow the title to be any JSX element or should we constraint it to be text? That would make styling easier.

Reply:
As of now, our component is minimalistic (e.g. we don't provide options for icon/control) so I think restricting title to just string makes sense